src/main/java/de/uni_tuebingen/ub/nppm/db/DeleteDB.java

```diff
@@ -0,0 +1,35 @@
+package de.uni_tuebingen.ub.nppm.db;
+
+import static de.uni_tuebingen.ub.nppm.db.AbstractBase.getSession;
+import org.hibernate.Session;
+import org.hibernate.query.NativeQuery;
+
+public class DeleteDB extends AbstractBase {
+
+    public static void updateAttribute(String table, String attribute, int id) throws Exception {
+        try ( Session session = getSession()) {
+            session.getTransaction().begin();
+            NativeQuery query = session.createNativeQuery("UPDATE " + table + " SET " + attribute + "= :value WHERE ID= :id");
+            query.setParameter("value", null);
+            query.setParameter("id", id);
+            query.executeUpdate();
+            session.getTransaction().commit();
+        }
+    }
+
+    public static String selectAttribute(String table, String attribute, int id) throws Exception {
```

Comment:
We should change the select... function into 2 functions that act in a similar way (see my comments related to the update... function).

```
public static Map<String,String> select(String table, int id, List<String> columns) {
    // ... slightly adjust previous implementation here
}

public static String select(String table, int id, String column) {
    Map<String, String> columnsAndValues = select(table, id, Arrays.asList(column));
    return columnsAndValues[column];
}
```

Reply:
Done look at commit: problem DeleteDB solved